WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueSocialFragment.java

```diff
@@ -276,6 +304,22 @@ protected boolean changedUsername() {
         return !StringUtils.equals(getArguments().getString(ARG_USERNAME), mUsername);
     }
 
+    protected void launchDialog() {
+        final Bundle extras = new Bundle();
+        extras.putString(UsernameChangerFullScreenDialogFragment.EXTRA_DISPLAY, mEditTextDisplayName.getText().toString());
```

Comment:
What do you think about introducing a "newInstance" type of method to hide away the setup of the input bundle? For example, a `UsernameChangerFullScreenDialogFragment.newInputBundle()` static method could accept the displayName and the username as parameters and return a properly populated bundle ready to be passed to `.setContent()` below.

Reply:
I added the `newBundle` method in 153764450605592589efd5ee43bda906cc39e766.